contracts/PetPark.sol

```diff
@@ -1,7 +1,100 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    enum AnimalType {
+        None,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+
+    enum Gender {
+        Male,
+        Female
+    }
+
+    struct Borrower {
+        uint age;
+        uint gender;
```

Comment:
The gender should be of type `Gender` as you have created an enum for it

Reply:
but i dont see how we can use a uint input to verify if the gender is male or female..
is there a way to index an enum based on the given uint input?